Source/WebGPU/WGSL/TypeCheck.cpp

```diff
@@ -30,11 +30,9 @@
 #include "ASTStringDumper.h"
 #include "ASTVisitor.h"
 #include "ContextProviderInlines.h"
+#include "TypeStore.h"
```

Comment:
The type store is where you go shopping for all the types you need to buy

Reply:
Lol, I think maybe that's a fronted/react/redux thing? Should I rename it?